dynamic.tex

```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
 </html>
 \end{minted}
 
+
+\begin{aside}{Installing React and ReactDOM}
+
+Now that we've made a project folder using \texttt{npm init},
+and we have a \texttt{project.json} file,
+we can install our first modules:
```

Comment:
do we tell people anywhere that they have to go into the newly-created project directory before running `npm install`?